### PR TITLE
Update agile.md

### PR DESCRIPTION
**Pull request by gabbylisper**
Editing completed; 1 hour spent.

### CODE REVIEW

File: _drafts/agile.md [hunk 1]
```diff
@@ -190,20 +189,20 @@ Not with the lack of all that.
 **Principle #12**:
 "At regular intervals, the team reflects on how to become more effective, then tunes and adjusts its behavior accordingly."
 
-That's a great principle, which is translated into so-called retrospective
-meetings. They work just fine, provided the decisions made at them are
-making the team better. Unfortunately, in most cases, programmers in Agile teams are
+That's a great principle which is translated into so-called retrospective
+meetings. They work just fine s long as decisions
```
Comment (yegor256) [inline _drafts/agile.md]: @gabbylisper typo?

File: _drafts/agile.md [hunk 2]
```diff
@@ -15,96 +15,96 @@ keywords:
...
+misunderstand them. Consequently, here is a summary of what's going on and my
+interpretation of each principle.
 
 <!--more-->
 
 **Principle #1**:
 "Our highest priority is to satisfy the customer through early and continuous delivery of valuable software."
 
-Staying focused on "satisfy the customer" Agile adepts totally forget about
+By focusing on "satisfy the customer", Agile adepts totally forgets about
```
Comment (yegor256) [inline _drafts/agile.md]: @gabbylisper are you sure it's `forgets`? Seems like "Agile adepts forget", not "forgets". or I'm wrong?

File: _drafts/agile.md [hunk 3]
```diff
@@ -15,96 +15,96 @@ keywords:
...
+ the Manifesto means. We make sure that our process is
+"early and continuous", which will result to customer satisfaction. We focus on improving
+our process, not satisfying the customer. Satisfaction is the consequence, not
 the primary objective.
 
 {% quote If the customer is not satisfied, we find another customer %}
 
 **Principle #2**:
-"Welcome changing requirements, even late in development. Agile processes harness change for the customer's competitive advantage."
+"Welcome change requirements, even late in development. Agile processes harness change for the customer's competitive advantage."
```
Comment (yegor256) [inline _drafts/agile.md]: @gabbylisper we should not change that, this text comes from the Manifesto itself. I'm just quoting it here

File: _drafts/agile.md [hunk 4]
```diff
@@ -15,96 +15,96 @@ keywords:
...
+In this case, any change will be welcome, since it won't affect anything.
 There simply won't be anything to affect. But this is not what the Manifesto
 means! This principle means that our requirements management process is so
-**powerful** that it can accept changes at any moment. It's rather difficult to
+**powerful** that it can accept change at any moment. However, it's rather difficult to
 achieve, if requirements are actually documented.
 
 **Principle #3**:
-"Deliver working software frequently, from a couple of weeks to a couple of months, with a preference to the shorter timescale."
+"Frequently deliver working software, from a couple of weeks to a couple of months, with a preference to the shorter timescale."
```
Comment (yegor256) [inline _drafts/agile.md]: @gabbylisper same here, we should not change this sentence

**Comment by yegor256** [comment]
@gabbylisper thanks!

File: _drafts/agile.md (hunk 1 above)
Comment (gabbylisper) [inline _drafts/agile.md]: yes; should be "as long as"

File: _drafts/agile.md (hunk 4 above)
Comment (gabbylisper) [inline _drafts/agile.md]: Noted. Should remain as it is.

File: _drafts/agile.md (hunk 3 above)
Comment (gabbylisper) [inline _drafts/agile.md]: Agreed.

File: _drafts/agile.md (hunk 2 above)
Comment (gabbylisper) [inline _drafts/agile.md]: No, you're right. Should be "Agile adepts forget". Though in this case,  I'm still rooting for "Agile experts..."

**Pull request merged**